client/src/constants/AppConstants.js

```diff
@@ -1,5 +1,6 @@
 export default {
-  REGISTER_USER: 'REGISTER_USER',
-  LOGIN_USER: 'LOGIN_USER',
-  CREATE_GROUP: 'CREATE_GROUP'
+  SET_USER: 'SET_USER',
+  CREATE_GROUP: 'CREATE_GROUP',
+  SET_GROUP: 'SET_GROUP',
+  SET_GROUP_MESSAGE: 'SET_GROUP_MESSAGE'
```

Comment:
Consider using npm module `key-mirror` here. It will avoid you having to duplicate the same name in both the key and value

Reply:
Link to keymirror: https://www.npmjs.com/package/keymirror